timon/probes/probes.py

```diff
@@ -270,7 +270,7 @@ def __repr__(self):
 
 class ThreadProbe(Probe):
     async def probe_action(self):
-        print("THREAD")
+        logger.debug("THREAD")
```

Comment:
not sure we really need this. perhaps just remove it?

Reply:
deleted